mkdocs.yml

```diff
@@ -11,47 +11,50 @@ pages:
 - ['index.md', 'Overview']
 
 - ['index.md', 'API', 'Using the API']
-- ['kb/api-version-3/object-field-references.md', 'API', 'Object Field References']
-- ['kb/api-version-3/error-codes-messages.md', 'API', 'Error Codes & Messages']
-
-- ['kb/api-version-3/campaigns.md', 'Methods', 'Campaigns']
-
-- ['kb/api-version-3/custom-fields.md', 'Methods', 'Custom Fields']
-
-- ['kb/api-version-3/custom-redirects.md', 'Methods', 'Custom Redirects']
-
-- ['kb/api-version-3/dynamic-content.md', 'Methods', 'Dynamic Content']
-
-- ['kb/api-version-3/emails.md', 'Methods', 'Email']
-
-- ['kb/api-version-3/email-templates.md', 'Methods', 'Email Templates']
-
-- ['kb/api-version-3/forms.md', 'Methods', 'Forms']
-
-- ['kb/api-version-3/lifecycle-histories.md', 'Methods', 'Lifecycle Histories']
-
-- ['kb/api-version-3/lifecycle-stages.md', 'Methods', 'Lifecycle Stages']
-
-- ['kb/api-version-3/lists.md', 'Methods', 'Lists']
-
-- ['kb/api-version-3/list-memberships.md', 'Methods', 'List Memberships']
-
-- ['kb/api-version-3/opportunities.md', 'Methods', 'Opportunities']
-
-- ['kb/api-version-3/prospects.md', 'Methods', 'Prospects']
-
-- ['kb/api-version-3/prospect-accounts.md', 'Methods', 'Prospect Accounts']
-
-- ['kb/api-version-3/tags.md', 'Methods', 'Tags']
-
-- ['kb/api-version-3/tag-objects.md', 'Methods', 'TagObjects']
-
-- ['kb/api-version-3/users.md', 'Methods', 'Users']
-
-- ['kb/api-version-3/visitors.md', 'Methods', 'Visitors']
-
-- ['kb/api-version-3/visitor-activities.md', 'Methods', 'Visitor Activities']
-
-- ['kb/api-version-3/visits.md', 'Methods', 'Visits']
+- ['kb/object-field-references.md', 'API', 'Object Field References']
+- ['kb/error-codes-messages.md', 'API', 'Error Codes & Messages']
```

Comment:
I moved `kb/error-codes-messages.md` and `kb/object-field-references.md` here out of the `kb/v3` folder. This allows the v3 <-> v4 linking object to work, since those two pages probably shouldn't be version-specific